### PR TITLE
Add method to determine whether CSG1 is loaded, fixes #5353

### LINKED ISSUE #5353: Add method to check if CSG1 is loaded
I want to use the booster sprites from CSG1.DAT for the Looping and Corkscrew RC. Since not everybody has RCT1, a simple method to check if CSG1 is loaded should be added to check this. This will also be useful in many other cases where CSG1 sprites are needed.

### PR DESCRIPTION
**Pull request by Gymnasiast**
diamond heights 2017-04-11 20-00-38

### CODE REVIEW

**Comment by Gymnasiast** [comment]
I get this error:
```
CMakeFiles/testpaint.dir/src/openrct2/ride/coaster/looping_roller_coaster.c.o: In function `looping_rc_track_booster':

looping_roller_coaster.c:(.text+0x1a3e0): undefined reference to `is_csg_loaded'

CMakeFiles/testpaint.dir/src/openrct2/ride/coaster/corkscrew_roller_coaster.c.o: In function `corkscrew_rc_track_booster':

corkscrew_roller_coaster.c:(.text+0x184bf): undefined reference to `is_csg_loaded'
```
@marijnvdwerf

**Comment by marijnvdwerf** [comment]
That just means the `is_csg_loaded` method doesn't exist in the `testpaint` target. You need to add an implementation of it to https://github.com/OpenRCT2/OpenRCT2/blob/develop/test/testpaint/compat.c, which simply returns `false`.

File: test/testpaint/compat.c [hunk 1]
```diff
@@ -165,3 +165,8 @@ bool track_element_is_inverted(rct_map_element *trackElement)
 {
 	return trackElement->properties.track.colour & TRACK_ELEMENT_COLOUR_FLAG_INVERTED;
 }
+
+bool is_csg_loaded()
+{
+	return false;
+}
```
Comment (marijnvdwerf) [inline test/testpaint/compat.c]: Missing newline

Comment (Gymnasiast) [inline test/testpaint/compat.c]: Now fixed.

**Pull request merged**